### DISCUSSION TITLE
can't sync data from binlog load

### DISCUSSION THREAD

**Discussion by fengyonghui** [apache/doris#13624] [A - General / Q&A] [answered]
trying to binlog sync data from RDS, got warning message:
2022-10-25 11:33:33,526 WARN (thrift-server-pool-3|242) [FrontendServiceImpl.loadTxnRollback():1028] failed to rollback txn 1012: errCode = 2, detailMessage = transaction not found

what's wrong?
thx

**Comment by hf200012** [comment, marked as answer]
There are still some problems with binlog load. We do not recommend using it in the production environment. This method will also put pressure on fe. We recommend that you use Flink CDC + Doris Flink Connector, which can be used to implement Exactly-once or use Flink. The ability to cluster for better throughput

**Comment by yclxiao** [comment]
how to solve this problem
